### PR TITLE
Feature/1654 delete subs v2

### PR DESCRIPTION
**Pull request by crbrox**
Implements #1654 

@fgalan 
@kzangeli

### CODE REVIEW

**Comment by fgalan** [comment]
LGTM (passing the ball to @kzangeli for final LGMT + merge)

File: src/lib/serviceRoutinesV2/deleteSubscription.cpp [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  HttpStatusCode  scc = sc.code;
+
+  if (scc != SccOk)
+  {
+    OrionError oe;
+
+    ciP->httpStatusCode = scc;
+    oe.code             = scc;
+    oe.reasonPhrase     = sc.reasonPhrase;
+
```
Comment (kzangeli) [inline src/lib/serviceRoutinesV2/deleteSubscription.cpp]: So many lines about StatusCode ... it's a little confusing.

Could be replaced with something like this:

    if (uncr.statusCode.code != SccOk)
   {
      OrionError oe(uncr.statusCode);
 
Not important, just an idea ...

Comment (crbrox) [inline src/lib/serviceRoutinesV2/deleteSubscription.cpp]: Fixed b4ec1bd

**Comment by kzangeli** [comment]
LGTM

**Pull request merged**